### PR TITLE
Add gallery to data view controller

### PR DESCRIPTION
**Pull request by IvanJanuary**

### CODE REVIEW

File: PersonDataRequest/PersonDataRequest.xcodeproj/xcshareddata/xcschemes/PersonDataRequest.xcscheme [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Lana-Resquit) [inline PersonDataRequest/PersonDataRequest.xcodeproj/xcshareddata/xcschemes/PersonDataRequest.xcscheme:1]: I believe this file should not be here

File: PersonDataRequest/PersonDataRequest/Controller/DataViewController.swift [hunk 2]
```diff
@@ -11,13 +11,20 @@ class DataViewController: UIViewController {
     
     var selectedName: String?
     
+    var gallery: Gallery = Gallery()
```
Comment (Lana-Resquit) [inline PersonDataRequest/PersonDataRequest/Controller/DataViewController.swift:14]: [Question] Why do you need this Gallery class? Does it give you benefits? Could you just add an array of pictures here?

File: PersonDataRequest/PersonDataRequest/PictureCell/PictureCell.xib [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Lana-Resquit) [inline PersonDataRequest/PersonDataRequest/PictureCell/PictureCell.xib:1]: This xib is not useful anymore, I gues it should be removed

Comment (IvanJanuary) [reply PersonDataRequest/PersonDataRequest/PictureCell/PictureCell.xib:1]: Removed

File: PersonDataRequest/PersonDataRequest.xcodeproj/xcshareddata/xcschemes/PersonDataRequest.xcscheme (hunk 1 above)
Comment (IvanJanuary) [reply PersonDataRequest/PersonDataRequest.xcodeproj/xcshareddata/xcschemes/PersonDataRequest.xcscheme:1]: Removed

**Pull request merged**